### DISCUSSION TITLE
pfSense V2.6 just released

### DISCUSSION THREAD

**Discussion by nickh66** [travisghansen/hass-pfsense#65] [General]
Hi all,
I was wondering if any one has upgraded their pfsense box to the newly released v2.6.  I'm curious if there are any complications found with this HA integration after the fireware upgrade?
Anything we should be aware of before upgrading?
Cheers
Nick

**Comment by travisghansen** [comment]
I’ve upgraded an instance and haven’t seen any errors yet. A couple (useless) datapoints for some mount points appear to no longer be returned from certain methods but otherwise everything looks clean so far.

**Reply by nickh66** [reply]
Good to know.  Thanks for the info.
